website/src/roadmap/portability.md

```diff
@@ -18,10 +19,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Portability Framework
-
-* TOC
-{:toc}
+# Portability Framework Roadmap
```

Comment:
@herohde please be candid in your thoughts about how I cut your writing in half and put half here, half on the wiki

Reply:
http://apache-beam-website-pull-requests.storage.googleapis.com/6718/roadmap/portability/index.html